providers/serializers.h

```diff
@@ -0,0 +1,102 @@
+/*
```

Comment:
I prefer calling these .inc files when they are just going to be embedded in another file.

Reply:
Renamed.